### ISSUE TITLE
Can't see Plugin Folder used every dependency

### ISSUE THREAD

**Issue by PanKaJxD69** [CarsCupcake/SkyblockRemake#39]
**Describe the bug**
Can't see my Plugin I used right Version and every dependency you told in ur video.

**Comment by PanKaJxD69** [comment]
Also That FastasyncWorldEdit also not working

**Comment by Chalityst** [comment]
this plugin working for me just install fastasyncworldedit without origin worldedit

**Comment by CarsCupcake** [comment]
yes you might have the wrong version of fawe installed or the wrong java version
